### DISCUSSION TITLE
Display number of comments on search results

### DISCUSSION THREAD

**Discussion by cluber22** [microsoft-search/pnp-modern-search#3690] [Q&A]
Hi,

Do I need to go down the route of creating some custom SPFx extension to be able to display the number of comments for each search result page? Much like on this forum when you search for something you can see the number of comments, it's quite a nice feature.

Is this on the road map anywhere? I don't mind having a go myself but will need a little prod in the right direction.

Thanks

**Comment by wobba** [comment]
Yes as comments are not part of the search results. See something like https://learn.microsoft.com/en-us/answers/questions/1196257/how-to-retrieve-the-comments-in-a-sharepoint-site
